anymaniax/orval
# Vue query improve reactive support

## Status



**READY/WIP/HOLD**

READY

## Description

Addresses: https://github.com/anymaniax/orval/issues/851

A few sentences describing the overall goals of the pull request's commits.

Improves vue reactivity support.

## Related PRs

List related PRs against other branches:

| branch              | PR       |
| ------------------- | -------- |
| other_pr_production | [link]() |
| other_pr_master     | [link]() |

## Todos


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.

See the sample update, I've expanded on the sample by dynamically updating the list to load in a single pet.

```bash
> git pull --prune
> git checkout <branch>
> grunt jasmine
```

1.
